src/main/java/io/skai/okta/internshipstreamingserviceparserxml/repository/impl/EpisodeAdditionalInfoRepository.java

```diff
@@ -54,10 +54,10 @@ public void save(EpisodeAdditionalInfo episodeAdditionalInfo) {
                 .fetch(getEpisodeAdditionalInfoRecordMapper());
     }
 
-    public EpisodeAdditionalInfo get(long id) {
+    public EpisodeAdditionalInfo get(long parent_id) {
```

Comment:
rename method to findByParentId + follow Java naming conversions (parent_id ---> parentId)